libs/ledger-live-common/src/families/xrp/walletApiAdapter.test.ts

```diff
@@ -16,6 +16,7 @@ describe("getWalletAPITransactionSignFlowInfos", () => {
 
       const expectedLiveTx: Partial<Transaction> = {
         ...xrpPlatformTx,
+        family: "xrp",
```

Comment:
Maybe we should have somewhere in Live common an object

```ts
export const defaultXRPTransaction = {
 family: "xrp"
}
```

And then do 

```ts
const expectedLiveTx: Partial<Transaction> = {
        ...xrpPlatformTx,
        ...defaultXRPTransaction,
}
```

That way we can easily extends the default values everywhere its needed